changelog.md

```diff
@@ -1,3 +1,8 @@
+# x.x.x
+
+* Made the internal CreationType/SnapshotType/Type official via the new InSnapshot/OutSnapshot/Instance/SnapshotOrInstance<typeof X>
```

Comment:
Maybe link to through the relevant documentation section above?

Reply:
done